app/views/admin/administrators/administrators/index.html.erb

```diff
@@ -1,3 +1,7 @@
+<% if @user_deleted %>
+  <%= govuk_panel title: "User Deleted", body: nil %>
+<% end %>
+
```

Comment:
Can we use the same pattern with flash messages as Ethan used in one of his other PRs?

Reply:
Removed - added flash